config/deploy.rb

```diff
@@ -9,17 +9,19 @@
 
 # Default deploy_to directory is /var/www/my_app
 set :deploy_to, "#{ENV['DEPLOY_DIR']}"
+set :param_file, "#{ENV['PARAM_FILE']}"
 
 # Set Rails environment
 # set, :rails_env, 'production'
 
-# Set RVM version
-set :rvm_ruby_version, '2.1.2'
+# Set rvm stuff
+set :rvm_ruby_version, File.read('.ruby-version').strip
```

Comment:
Just found out that dotenv-deployment has been deprecated so we should
switch over to dotenv-rails for everything. It shouldn't change anything,
just need to update the Gemfile / `bundle install`.